.travis.yml

```diff
@@ -1,15 +1,17 @@
 sudo: required
-dist: trusty
+dist: jammy
 language: node_js
 node_js:
-  - '12'
+  - '14.20.1'
 addons:
   chrome: stable
 
+services:
+  - xvfb
+
 before_install:
-  - export CHROME_BIN=chromium-browser
+  - export CHROME_BIN=/usr/bin/google-chrome
```

Comment:
why is this needed?

Reply:
Because of new versions of Ubuntu used in Travis